cocotb/share/include/vhpi_user.h

```diff
@@ -1122,6 +1122,14 @@ typedef enum
 #endif
 } vhpiSimControlT;
 
+/* arguments for vhpiStop or vhpiFinish */
+typedef enum
+{
+  vhpiDiagNone          = 0, /* prints nothing */
+  vhpiDiagTimeLoc       = 1, /* prints simulation time and location */
+  vhpiDiagTimeLocCPUMem = 2  /* prints simulation time, location and statistics about CPU and memory usage */
+} vhpiDiagT;
+
```

Comment:
If these are symbols we invented that are not part of the standard vhpi header, then I do not think we should put them here. If they came from some non-standard extension of that header, can you link to its source?

Reply:
Yes, these headers should stay a copy of the ones in the standard. We have `vpi_user_ext.h` for VPI, could you add the VPI part there, and create a similar header for VHPI?

Reply:
Done